db/migrate/20240814150000_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
It will be nice to add a unique index to the email column since it's used to query users upon sign-in.

Reply:
By doing it on the validations as unique to true, does not work the same way?